lib/facter/self_service.rb

```diff
@@ -6,9 +6,46 @@
 
 Facter.add(:self_service, type: :aggregate) do
   confine kernel: 'Linux'
+  confine { Facter.value(:pe_build) }
 
   puppet_bin = '/opt/puppetlabs/bin/puppet'
 
+  configdir = if Facter.value(:os)['family'].eql?('RedHat') || Facter.value(:os)['family'].eql?('Suse')
+                '/etc/sysconfig'
+              else
+                '/etc/default'
+              end
+  postgresversion = if Facter.value(:os)['family'].eql?('Debian')
+                      "pe-postgresql#{Facter.value(:pe_postgresql_info)['versions'].first.first}"
```

Comment:
I was thinking about this one in #7. I figure there is a scenario where there may be multiple versions of PG installed. My logic may be flawed as I am assuming we will always use the newest.

With 14 shipping, we will probably have 11 and 14 installed at the same time with only one of them active. It seems like we should be using `pe_postgresql_info.installed_server_version` since that is what `puppet_enterprise` uses to determine if there is an upgrade necessary.  https://github.com/puppetlabs/puppet-enterprise-modules/blob/main/modules/pe_install/manifests/upgrade/postgres.pp#L75